docs/src/gettingstarted/introduction/template/cli-install.md

```diff
@@ -12,65 +12,30 @@ With all of the requirements met, install the CLI to start developing with Platf
 
 {{< asciinema src="videos/asciinema/verify-cli-extended.cast" >}}
 
-1. **Install the CLI**
+## 1. Install the CLI
 
-    In your terminal, run the following command depending on your operating system:
+{{% cli-installation %}}
 
-{{< codetabs >}}
+## 2. Clone your project locally
 
----
-title=Linux/macOS
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -fsS https://platform.sh/cli/installer | php
-
-<--->
-
----
-title=Windows
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -f https://platform.sh/cli/installer -o cli-installer.php
-php cli-installer.php
-
-{{< /codetabs >}}
-
-
-2. **Authenticate and Verify**
-
-   Once the installation has completed, you can run the CLI in your terminal with the command
-
-   ```bash
-   platform
-   ```
-
-   {{< note >}}
-
-   If you opened your free trial account using another login (such as GitHub),
-   you can't authenticate with this command until you setup your account password with Platform.sh in the console.
-
-   {{< /note >}}
+Get a list of all available commands with:
 
-   You should now see a list of your Platform.sh projects, including the template you made in this guide.
-   You can copy its *project ID* hash, and then download a local copy of the repository with this command:
+```bash
+platform list
+```
 
-   ```bash
-   platform get <project ID>
-   ```
+You should now see a list of your Platform.sh projects, including the template you made in this guide.
```

Comment:
:hammer_and_wrench: This command doesn't give a list of projects. It also seems out of the flow of the guide in general. I think `platform` makes more sense and should return what is stated here.

Reply:
I changed the flow a bit to a list, should be better fitting and clearer.